src/st/compatibility/class-wpml-page-builders-update.php

```diff
@@ -47,6 +47,16 @@ private function save_data( $post_id, $fields, $data ) {
 	 */
 	private function copy_meta_fields( $translated_post_id, $original_post_id, $meta_fields ) {
```

Comment:
@adrianosferreira This function should now be renamed to `copy_fields` and `$meta_fields` renamed to `$fields`

Reply:
@adrianosferreira Did you do this change?

Reply:
@brucepearson no, I missed that one, sorry. Fixing in a moment.